addon/-private/decorators/localized-attr.js

```diff
@@ -11,7 +12,22 @@ export default function (...args) {
   attrComputed.get = function () {
     const value = getter.call(this);
 
-    return value && value[this.getFieldLocale()];
+    if (!value) {
+      return;
+    }
+
+    const {
+      localizedModel: { showSomething = false, fallbacks = [] } = {}
+    } = getOwner(this).resolveRegistration('config:environment');
+
+    const keys = [this.getFieldLocale(), ...fallbacks];
+    let key = keys.find(key => Boolean(value[key]));
```

Comment:
Maybe lets add a quick note to the readme that this should not be used with number fields since we cast to Boolean here.

Reply:
Done